docs/user-guide/cli-using-integrating-apiml.md

```diff
@@ -40,53 +55,101 @@ Issue the following command:
+
+### Specifying a base path with Zowe V2 profiles
+
+Follow these steps:
+
+1. Note the complete path for a z/OSMF instance registered to API ML.
+
+    As an example, use the following path:
+
+    ```
+    https://myapilayerhost:port/ibmzosmf/api/v1
+    ```
 
-* `--base-path`: The base path of the API ML instance that you want to access.
-* `--disable-defaults`: Specify this flag to prevent default values from being stored in service profiles. If you do not use this flag, the defaults can override values in your base profile.
+    The format of base path can vary based on how API ML is configured at your site.
 
-**Note:** Ensure that you *do not* provide username, password, host, or port directly on the service commands or profiles. Supplying those options causes the CLI to ignore the API ML token in your base profile and directly access the service.
+2. To access that API ML instance, create a [service profile](../user-guide/cli-using-using-team-profiles#zowe-cli-profile-types) (or issue a command) with the `--base-path` value of `ibmzosmf/api/v1`. Your service profile uses the token and credentials stored in your default base profile.
 
-### Specifying a base path
+    Using the preceding example, you would issue the following command with your profile name:
 
-The following example illustrates a complete path for a z/OSMF instance registered to API ML. The format of base path can vary based on how API ML is configured at your site:
+    ```
+    zowe config set "profiles.myapimlprofile.properties.basepath" "ibmzosmf/api/v1" 
+    ```
 
-```
-https://myapilayerhost:port/ibmzosmf/api/v1
-```
+    Commands issued with this profile are routed through the layer to access an appropriate z/OSMF instance.
 
-To access that API ML instance, create a service profile (or issue a command) with the `--base-path` value of `api/v1`. Your service profile uses the token and credentials stored in your default base profile.
+    For an example, refer to the following command:
 
-```
-zowe profiles create zosmf myprofile123 --base-path ibmzosmf/api/v1 --disable-defaults
-```
+    ```
+    zowe plugins list --base-path "ibmzosmf/api/v1"
+    ```
 
-Commands issued with this profile are routed through the layer to access an appropriate z/OSMF instance.
+### Specifying a base path with Zowe V1 profiles
+
+Follow these steps:
+
+1. Note the complete path for a z/OSMF instance registered to API ML.
+
+    As an example, use the following path:
+
+    ```
+    https://myapilayerhost:port/ibmzosmf/api/v1
+    ```
+
+    The format of base path can vary based on how API ML is configured at your site.
+
+2. To access that API ML instance, create a [service profile](../user-guide/cli-using-using-profiles-v1#zowe-cli-v1-profile-types) (or issue a command) with the `--base-path` value of `ibmzosmf/api/v1`. Your service profile uses the token and credentials stored in your default base profile.
```

Comment:
Same as above. Change to:
Access the API ML instance by creating a ... (or issuing a command) ..."